src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,59 @@
+package domain;
+
+import static util.constant.Values.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+
+public class Lotto {
+
+  private List<Integer> numbers = new ArrayList<>();
+
+  public Lotto() {
+    createNumbers();
+  }
```

Comment:
상태 값이 한번 정해진 뒤 이후 수정이 없다면 꼭 `final`로 선언을 해주는 습관을 들여보시면 좋을 것 같아요. 

추가적으로 아래와 같이 생성자에서 상태 값을 직접 셋팅하기보다는 외부에서 모든 값을 셋팅한 뒤 생성자 파라미터를 통해서 전달해준다면 추후 테스트 코드 작성하기에도 더 쉬울거에요. 😃 

```suggestion
  private final List<Integer> numbers;

  public Lotto(List<Integer> numbers) {
    this.numbers = new ArrayList<>(numbers);
  }
```